app/services/ems_infra_dashboard_service.rb

```diff
@@ -125,12 +125,14 @@ def ems_utilization
   end
 
   def daily_provider_metrics
-    current_user = @controller.current_user
-    tp = TimeProfile.profile_for_user_tz(current_user.id, current_user.get_timezone) || TimeProfile.default_time_profile
+    @daily_provider_metrics ||= begin
+      current_user = @controller.current_user
+      tp = TimeProfile.profile_for_user_tz(current_user.id, current_user.get_timezone) || TimeProfile.default_time_profile
 
-    @daily_metrics ||= Metric::Helper.find_for_interval_name('daily', tp)
-                                     .where(:resource => (@ems || ManageIQ::Providers::InfraManager.all))
```

Comment:
If `@ems` is set here I feel like this shouldn't be an issue. The ems in the remote region and the one in the global should be separate objects. Are we sure that the bug isn't that we're pulling back all the metrics because `@ems` is nil for some reason?